packages/flutter/lib/src/material/material.dart

```diff
@@ -76,6 +85,15 @@ abstract class MaterialInkController {
 
 /// A piece of material.
 ///
+/// The Material widget is responsible for:
+/// 1. Clipping - Material clips its widget sub-tree to the shape specified by
+///               [type] and [borderRadius].
```

Comment:
i think markdown is going to want this to be indented by three rather than four or more, because four results in monospace font.

Reply:
done, though I have a feeling there will be another markdown iteration here :)